airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -353,7 +353,7 @@ class CustomerBalanceTransactions(BasePaginationStripeStream):
     name = "customer_balance_transactions"
 
     def path(self, stream_slice: Mapping[str, Any] = None, **kwargs):
-        return f"customers/{stream_slice['id']}/balance_transactions"
+        return f"customers/{stream_slice['record_id']}/balance_transactions"
```

Comment:
I don't think this will work. CustomerBalanceTransactions isn't subclass of the class that adds record_id.

Reply:
ah yes my bad